source/_integrations/ping.markdown

```diff
@@ -34,15 +34,20 @@ host:
   required: true
   type: string
 count:
-  description: Number of packets to send.
+  description: Number of packages to be sent up to a maximum of 100.
   required: false
   type: integer
   default: 5
 name:
   description: Let you overwrite the name of the device.
   required: false
   type: string
-  default: Ping Binary sensor
+  default: Ping Binary sensor [hostname]
+scan_interval:
```

Comment:
Please do not add/document `scan_interval`. It is already documented at [Entity component platform options](https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval). We decided not to add it to every component, because it's an advanced feature and in 95 % of the use cases not needed.